From e8453e147b92717403006bc6165f3c096b5550ad Mon Sep 17 00:00:00 2001
From: Franck Bui <franck.bui@mandriva.com>
Date: Wed, 5 Oct 2011 11:47:46 +0200
Subject: [PATCH 2/2] tools: don't rely on CONFIG_SYS_UBL_BLOCK being defined
 at compile time

Signed-off-by: Franck Bui <franck.bui@mandriva.com>
---
 tools/ublimage.h |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/tools/ublimage.h b/tools/ublimage.h
index c926689..7bd5ae1 100644
--- a/tools/ublimage.h
+++ b/tools/ublimage.h
@@ -28,8 +28,6 @@
 #ifndef _UBLIMAGE_H_
 #define _UBLIMAGE_H_
 
-#include <config.h>
-
 #if !defined(CONFIG_SYS_UBL_BLOCK)
 #define CONFIG_SYS_UBL_BLOCK 512
 #endif
-- 
1.7.5.1

