diff -Naur u-boot-2021.04-rc4/include/config_distro_bootcmd.h u-boot-2021.04-rc4.tpg/include/config_distro_bootcmd.h
--- u-boot-2021.04-rc4/include/config_distro_bootcmd.h	2021-03-15 16:06:41.000000000 +0000
+++ u-boot-2021.04-rc4.tpg/include/config_distro_bootcmd.h	2021-03-18 18:33:04.261454458 +0000
@@ -151,7 +151,8 @@
 		"load ${devtype} ${devnum}:${distro_bootpart} "           \
 			"${fdt_addr_r} ${prefix}${efi_fdtfile}\0"         \
 	\
-	"efi_dtb_prefixes=/ /dtb/ /dtb/current/\0"                        \
+	"efi_dtb_prefixes=/ /dtb/ /dtb/current/ "                         \
+		"/boot/ /boot/dtb/ /boot/dtb/current/\0"                  \
 	"scan_dev_for_efi="                                               \
 		"setenv efi_fdtfile ${fdtfile}; "                         \
 		BOOTENV_EFI_SET_FDTFILE_FALLBACK                          \
